### PR TITLE
[New] Add quasar-framework@0.15.10 w/ npm auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #11985

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/quasarframework/quasar
 * Official website (optional, not the repository):  http://quasar-framework.org
 * NPM package url (optional): https://www.npmjs.com/package/quasar-framework
 * License and its reference: [MIT](https://github.com/quasarframework/quasar/blob/dev/LICENSE)
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 230
   - Count of stars: 4128
   - Count of forks: 364
 * NPM download stats (optional):
   - Downloads in the last day: 277
   - Downloads in the last week: 1561
   - Downloads in the last month: 7923

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
Pattern `**/!(MaterialIcons-Regular*|*.*(styl|gz))` setting explanation:
1. `MaterialIcons-Regular*` is to exclude app-additions/assets/fonts/icons/MaterialIcons-Regular.woff in old versions. It has been hosted by cdnjs https://cdnjs.com/libraries/material-design-icons
2. `*.*(styl|gz)` is to exclude css source files and `quasar.standalone.min.js.gz`

**Comment by extend1994** [comment]
Wait https://github.com/quasarframework/quasar/issues/778

**Comment by extend1994** [comment]
Waiting v0.15 for official standalone build. See https://github.com/quasarframework/quasar/issues/778#issuecomment-351992510.

**Comment by sashberd** [comment]
@extend1994 I think v0.15 [was released](https://github.com/quasarframework/quasar/tree/v0.15.0)

**Comment by extend1994** [comment]
@sashberd Thanks for reminding!
1. Updated to 0.15.10, `filename` is according to https://github.com/quasarframework/quasar/blob/dev/dev-umd/index.mat.umd.html#L67
2. Only need to host 0.15+, refer https://github.com/quasarframework/quasar/issues/778#issuecomment-351992510

@sashberd @sufuf3 Please help review it, thanks!

**Comment by extend1994** [comment]
@sashberd According to https://github.com/quasarframework/quasar/blob/dev/build/script.build.javascript.js#L74-L75 & https://github.com/quasarframework/quasar/blob/dev/build/script.build.javascript.js#L87-L111, I think icons & i18n should be included in the files under `dist/umd`.

File: ajax/libs/quasar-framework/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "vuejs",
+    "vue",
+    "quasar",
+    "js",
+    "phone",
+    "tablet",
+    "desktop",
+    "spa",
+    "website",
+    "electron"
```
Comment (sufuf3) [inline ajax/libs/quasar-framework/package.json]: Maybe we can add more keywords from repo topic, what do you think?
Like ios, android, electron, material, vue-components, material-theme, material-design, material-components ...
Thanks

**Comment by extend1994** [comment]
@sufuf3 Updated, please help check it, thanks.

**Comment by PeterDaveHello** [comment]
@rstoenescu we're going to merge this pull request! There is no action needed, but you can consider to let the users know that this library is now on CDNJS!

**Comment by rstoenescu** [comment]
Thank you all! Especially for @extend1994 taking time to do a PR.

**Comment by extend1994** [comment]
@rstoenescu No problem, my pleasure 👍

**Pull request merged**